src/shared/scrapers/USA/MO/index.js

```diff
@@ -274,8 +274,9 @@ const scraper = {
           unassigned.cases += parse.number(countyData.Cases || 0);
           unassigned.deaths += parse.number(countyData.Deaths || 0);
         } else {
-          countyName = geography.addCounty(countyName);
-
+          if (countyName.toUpperCase().indexOf(' CITY') === -1) {
+            countyName = geography.addCounty(countyName);
+          }
```

Comment:
Looks like we need a special case for the city -- can you return an object with `{ city: "St Louis", county: "Whatever it's supposed to be", cases...}` for the city?

Reply:
It is my understanding that St. Louis City is the county name.  This is true for many US cities such as my hometown Richmond City, VA (not to be confused with it's neighboring county Richmond County, VA)
https://en.wikipedia.org/wiki/St._Louis_County,_Missouri
"St. Louis County borders, but does not include, the city of St. Louis, which is an independent city."

Reply:
Oh... So there is a separate county called `St. Louis County` and another one called `St. Louis City`?

Nice, that makes a lot of sense smh lol